newIDE/app/src/MainFrame/EditorContainers/HomePage/index.js

```diff
@@ -264,6 +265,7 @@ export const HomePage = React.memo<Props>(
         setPreCreationDialogOpen(false);
         setOutputPath(computeDefaultProjectPath());
         onOpenProjectAfterCreation({ project, storageProvider, fileMetadata });
+        setIsOpening(false);
```

Comment:
You should also set this to false if there is an early return (this could happen if the project can't be created for some reasons, like can't create the path on desktop).